pubtools/_quay/operator_pusher.py

```diff
@@ -13,6 +13,7 @@
     log_step,
     get_pyxis_ssl_paths,
 )
+from .quay_client import QuayClient
```

Comment:
Is this necessary?

Reply:
yeah, QuayClient.MANIFEST_LIST_TYPE is used for media_types